zipkin-thrift/src/main/thrift/com/twitter/zipkin/zipkinCore.thrift

```diff
@@ -220,16 +242,28 @@ enum AnnotationType { BOOL, BYTES, I16, I32, I64, DOUBLE, STRING }
  * you can see the different points of view, which often help in debugging.
  */
 struct BinaryAnnotation {
+  /**
+   * Name used to lookup spans, such as "http.uri" or "finagle.version".
+   */
   1: string key,
+  /**
+   * Serialized thrift bytes, in TBinaryProtocol format.
```

Comment:
@yurishkuro ps I finally hunted down what that endian stuff was about!! little-endian wasn't added until thrift 0.9.3!